network/server.go

```diff
@@ -79,10 +85,39 @@ type Peer struct {
 	Info peer.AddrInfo
 }
 
+// setupLibp2pKey is a helper method for setting up the networking private key
+func setupLibp2pKey(secretsManager secrets.SecretsManager) (crypto.PrivKey, error) {
+	var key crypto.PrivKey
+	if secretsManager.HasSecret(secrets.NetworkKey) {
+		// The key is present in the secrets manager, read it
+		networkingKey, readErr := ReadLibp2pKey(secretsManager)
+		if readErr != nil {
+			return nil, fmt.Errorf("unable to read networking private key from SM, %v", readErr)
```

Comment:
Avoid introducing new project-scope acronyms such as SM, feel free to write out the entire name: secretManager.

Also, look for other usages where `sm` is the part of the variable name, don't just focus on these string occurrences

Reply:
Fixed 👍 

ded3f1e